content/en/serverless/datadog_lambda_library/extension.md

```diff
@@ -28,20 +32,28 @@ The Datadog Extension is distributed as its own Lambda Layer (separate from the
     ```
 
     Replace the placeholder `AWS_REGION` in the Lambda Layer ARN with appropriate values.
+    
+    Note that this Layer is separate from the Datadog Lambda Library. If you installed the Datadog Lambda Library as a Lambda Layer,
+    your function should now have two Lambda Layers attached.
 
-3. If you are using Node.js or Python, add the Lambda Layer for the [Datadog Library][7] to your AWS Lambda function:
+3. Reference the [sample code][10] to submit a custom metric.
 
-    ```
-    arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-<RUNTIME>:<VERSION>
-    ```
+### As a container image
+
+If your function is deployed as a container image, you cannot add Lambda Layers to your function. Instead, you must directly install the Datadog Lambda Library and the Datadog Lambda Extension in your function's image.
+
+First, install the Datadog Lambda Library by following the installation instructions for [Node.js][5] or [Python][4]. Be sure to use the installation instructions specifically for functions deployed as container images.
 
-    The available `RUNTIME` options are `Node10-x`, `Node12-x`, `Python37`, and `Python38`. For `VERSION`, see the latest release for [Node.js][8] or [Python][9].
+Then, add the Datadog Lambda Extension to your container image by adding the following to your Dockerfile:
 
-4. Reference the [sample code][10] to submit a custom metric.
+```
+WORKDIR /opt/extensions
+COPY --from=datadog/lambda-extension:6 /opt/extensions/ .
```

Comment:
Is there a version of this that doesn't change the workdir? It could change the behaviour of the rest of the docker file if the user forgets to reset the dir.

Reply:
Good point 👍